Microsoft.Dynamics365.UIAutomation.Api.UCI/DTO/AppElementReference.cs

```diff
@@ -418,7 +418,7 @@ public static class AppElements
             { "Entity_RecordSetNavList", "//ul[contains(@data-id, 'recordSetNavList')]" },
             { "Entity_RecordSetNavCollapseIcon", "//*[contains(@data-id, 'recordSetNavCollapseIcon')]" },
             { "Entity_RecordSetNavCollapseIconParent", "//*[contains(@data-id, 'recordSetNavCollapseIcon')]" },
-            { "Entity_TabList", "//ul[contains(@id, \"tablist\")]" },
+            { "Entity_TabList", ".//ul[contains(@id, \"tablist\")]" },

```

Comment:
Can you confirm that Entity.SelectTab() still functions with this change?

Reply:
Yes with this change it works well with both main page tab and dialog tab , i have already tested this